### PR TITLE
games-board/xmahjongg: Fix C++17 does not allow register storage class

### PR DESCRIPTION
**Pull request by listout**
Closes: https://bugs.gentoo.org/895896

### CODE REVIEW

File: games-board/xmahjongg/xmahjongg-3.7-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools desktop
+
+DESCRIPTION="friendly GUI version of xmahjongg"
```
Comment (mm1ke) [inline games-board/xmahjongg/xmahjongg-3.7-r2.ebuild:8]: Capital first letter, `DESCRIPTION="Friendly GUI version of xmahjongg"`

File: games-board/xmahjongg/xmahjongg-3.7-r2.ebuild [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+HOMEPAGE="http://www.lcdf.org/xmahjongg/"
+SRC_URI="http://www.lcdf.org/xmahjongg/${P}.tar.gz"
```
Comment (mm1ke) [inline games-board/xmahjongg/xmahjongg-3.7-r2.ebuild:10]: I suggest: http -> https for `HOMEPAGE` and `SRC_URI`

File: games-board/xmahjongg/xmahjongg-3.7-r2.ebuild (hunk 1 above)
Comment (chewi) [reply games-board/xmahjongg/xmahjongg-3.7-r2.ebuild:8]: I handled it.

File: games-board/xmahjongg/xmahjongg-3.7-r2.ebuild (hunk 2 above)
Comment (chewi) [reply games-board/xmahjongg/xmahjongg-3.7-r2.ebuild:10]: I handled it.

**Pull request closed without merging**